### PR TITLE
Add crowdin localization details to README and strings.xml

### PR DESCRIPTION
**Pull request by ThomDietrich**
Related to #342 
Related to #583 

Signed-off-by: Thomas Dietrich <thomas.dietrich@tu-ilmenau.de> (github: ThomDietrich)

### CODE REVIEW

**Comment by mueller-ma** [comment]
XML comments are preserved by crowdin, so can you add one to values/strings.xml?

**Comment by ThomDietrich** [comment]
I guess so: https://github.com/openhab/openhab-android/pull/583/files#diff-376838b5b4d4d7e9f619051b68ac437d

Let's find out :)

File: README.md [hunk 1]
```diff
@@ -21,6 +21,17 @@ openHAB Android application is a native client for openHAB. It uses REST API of
 
 <img src="https://raw.githubusercontent.com/openhab/openhab.android/master/fastlane/metadata/android/en-US/phoneScreenshots/main_menu.png" width="200px"> <img src="https://raw.githubusercontent.com/openhab/openhab.android/master/fastlane/metadata/android/en-US/phoneScreenshots/widget_overview.png" width="200px"> <img src="https://raw.githubusercontent.com/openhab/openhab.android/master/fastlane/metadata/android/en-US/phoneScreenshots/astro_binding.png" width="200px">
 
+## Localization
+
+Concerning all `values-*` folders at [mobile/src/main/res](mobile/src/main/res/)
+
+All language/regional translations are managed with [Crowdin](https://crowdin.com/).
+Please do NOT contribute translations as Pull Requests against the `mobile/src/main/res/values-*` folders directly but submit them through the Crowdin webservice:
```
Comment (mueller-ma) [inline README.md]: pull requests?

Comment (mueller-ma) [reply README.md]: directly, but?

Comment (mueller-ma) [reply README.md]: `web service`?

Comment (ThomDietrich) [reply README.md]: jup

File: README.md [hunk 2]
```diff
@@ -23,6 +23,8 @@ openHAB Android application is a native client for openHAB. It uses REST API of
 
 ## Localization
 
+[![Crowdin](https://d322cqt584bo4o.cloudfront.net/openhab-android/localized.svg)](https://crowdin.com/project/openhab-android)
```
Comment (mueller-ma) [inline README.md]: It looks better, when you put it to the other badges in the first line.

**Comment by ThomDietrich** [comment]
@mueller-ma I've also added the crowdin badge to the header and the localization chapter.

Side note: you could add other badges as well. There are a few noteworthy over at https://shields.io (not sure if any or which are meaningful)

**Comment by mueller-ma** [comment]
I had a look at this page some time ago and didnt find any usefull badges except bountysource.

I would avoid having the badge two times.

**Pull request merged**